scripts/data/wikidata_us_states.csv

```diff
@@ -0,0 +1,293 @@
+Name,ID
+maine,Q724
+maine; united states,Q724
+me,Q724
+state of maine,Q724
+new hampshire,Q759
+nh,Q759
+the granite state,Q759
+new hampshire; united states,Q759
+state of new hampshire,Q759
+the golden state,Q99
+california,Q99
+ca,Q99
+state of california,Q99
```

Comment:
At least with states we could do a manual filter on names. For example, instead of having `state of california` just have `california`.